src/components/__tests__/__snapshots__/calculate.test.js.snap

```diff
@@ -0,0 +1,32 @@
+// Jest Snapshot v1, https://goo.gl/fbAQLP
+
```

Comment:
![image](https://github.com/yusufobr/react-calculator/assets/93039359/c4f550de-f644-4e81-95b5-76a20b2234cd)

**[OPTIONAL]**

- [ ] It would be nice if you make your commit message capitalized to make your pull request more professional 👍